torchvision/prototype/transforms/_geometry.py

```diff
@@ -172,15 +172,21 @@ class FiveCrop(Transform):
         torch.Size([5])
     """
 
-    _transformed_types = (features.Image, PIL.Image.Image, features.is_simple_tensor)
+    _transformed_types = (features.Image, PIL.Image.Image, features.is_simple_tensor, features.Video)
 
     def __init__(self, size: Union[int, Sequence[int]]) -> None:
         super().__init__()
         self.size = _setup_size(size, error_msg="Please provide only two dimensions (h, w) for size.")
 
     def _transform(
-        self, inpt: features.ImageType, params: Dict[str, Any]
-    ) -> Tuple[features.ImageType, features.ImageType, features.ImageType, features.ImageType, features.ImageType]:
+        self, inpt: features.ImageOrVideoType, params: Dict[str, Any]
+    ) -> Tuple[
+        features.ImageOrVideoType,
+        features.ImageOrVideoType,
+        features.ImageOrVideoType,
+        features.ImageOrVideoType,
+        features.ImageOrVideoType,
+    ]:
```

Comment:
_Feel free to ignore if mypy is happy_

This is not accurate. We don't have `Tuple[features.ImageOrVideoType, ...]` here, but rather `Union[Tuple[features.ImageType],...], Tuple[features.VideoType, ...]]`. Meaning, the type will not vary inside the returned tuple. We either get a tuple of images or a tuple of videos.

Reply:
This is why IMO we should avoid `features.ImageOrVideoType` and instead define it as `Union[Image, Video]`. Since this will be fixed in a follow up, there is no point messing with mypy (which is happy) here. I'll implement this on a follow up.